src/test/java/homework_jdi/Steps.java

```diff
@@ -0,0 +1,63 @@
+package homework_jdi;
+
+import homework_jdi.commonelements.HeaderMenuItems;
+import org.testng.Assert;
+
+import static homework_jdi.pages.homepage.entities.JdiUser.ROMAN;
+import static java.lang.String.format;
+public class Steps {
+
+    public Steps() {
+    }
+
+    public void openJsiSite() {
+        JdiSite.open();
+    }
+
+    public void loginWasSuccessful() {
+        JdiSite.login(ROMAN);
+        String actualUserName = JdiSite.jdiHomePage.getUserName();
+        Assert.assertEquals(actualUserName, ROMAN.getFullName(),
+                format("Expected loggined user: %s, but get %s", ROMAN, actualUserName));
+    }
+
+    public void clickHeaderMenu(HeaderMenuItems metalsAndColors) {
+        JdiSite.clickItemOfHeaderMenu(metalsAndColors);
+    }
+
+    public void pageIsOpened(HeaderMenuItems item) {
+        switch (item) {
+            case METALS_AND_COLORS:
+                JdiSite.metalColorsPage.checkOpened();
+                break;
+        }
+    }
+
+    public void fillPageWithDataFromJson(DataFromJsonForMetalsColorPage dataFromJsonForMetalsColorPage) {
+        JdiSite.metalColorsPage.refresh();
```

Comment:
Calls below should not be here!
Take a look at the classwork and submit/login/fill form's method.

Reply:
fix